### DISCUSSION TITLE
RAM with Docker for Windows

### DISCUSSION THREAD

**Discussion by TangJames** [thijsvanloef/palworld-server-docker#74] [Q&A]
It looks like on my docker for windows my RAM is capped at 16GB but my computer has 32GB. How do I change this to allocate more RAM to my containers?

**Comment by TangJames** [comment]
Okay I realized that docker desktop is capped at 16GB of RAM usage and there's no way around that. This should be included in the startup for people to know.

**Comment by thijsvanloef** [comment]
@TangJames If you are using Docker Desktop for Windows with the WSL integration, by default allows Docker to use 50% of RAM, however you can change that by changing the WSL to use more resources: https://learn.microsoft.com/en-us/windows/wsl/wsl-config#configure-global-options-with-wslconfig

**Reply by TangJames** [reply]
I've done this config for the wslconfig file but is it supposed to show on the docker desktop UI?
